lib/auth0/api/v2/organizations.rb

```diff
@@ -330,6 +330,52 @@ def delete_organizations_member_roles(organization_id, user_id, roles = [])
         end
         alias remove_organizations_member_roles delete_organizations_member_roles
 
+        # Get client grants associated to an organization
+        # @param organization_id [string] The Organization ID
+        # @param options [hash] The Hash options used to define the paging of results
+        #   * :client_id [string] The client_id of the client grant to retrieve.
+        #   * :audience [string] The audience of the client grant to retrieve.
+        #   * :per_page [integer] The amount of entries per page. Default: 50. Max value: 100.
+        #   * :page [integer] The page number. Zero based.
+        #   * :include_totals [boolean] True to include query summary in the result, false or nil otherwise.
+        def get_organizations_client_grants(organization_id, options= {})
+          raise Auth0::MissingOrganizationId, 'Must supply a valid organization_id' if organization_id.to_s.empty?
+          request_params = {
+            client_id:      options.fetch(:client_id, nil),
+            audience:       options.fetch(:audience, nil),
+            per_page:       options.fetch(:per_page, nil),
+            page:           options.fetch(:page, nil),
+            include_totals: options.fetch(:include_totals, nil)
+          }
+          path = "#{organizations_client_grants_path(organization_id)}"
```

Comment:
Is it necessary to interpolate the return value, if `organizations_client_grants_path(organization_id)` returns a string?

Reply:
Maybe not? Am just being consistent with the rest of the file https://github.com/auth0/ruby-auth0/pull/540/files/43bb301afee4d5ebd923eb897b53b998bc3465b6#diff-3936cbc470adbe60f39a627f01d4259b0c1121de790b4e7d3144c536f073ccf7R324